src/lib/djerba/plugins/tar/snv_indel/snv_indel_tools/Rscripts/supporting_functions.r

```diff
@@ -0,0 +1,392 @@
+preProcCNA <- function(segfile, genebed, gain, amp, htz, hmz, oncolist, genelist=NA){
+
+ # test
```

Comment:
test files should go in real tests (and I'm not even confident these files actually still exist)